host/traceh.c

```diff
@@ -224,7 +209,31 @@ void oe_log_message(bool is_enclave, oe_log_level_t level, const char* message)
                 _log_creation_failed_before = true;
                 return;
             }
-            _write_message_to_stream(log_file, is_enclave, level, message);
+
+            char* message_dup = strdup(message);
+            char* log_msg = strtok(message_dup, "[");
+            size_t len = strlen(log_msg);
```

Comment:
I think it could still be a good idea to log something like "Failed to apply custom formatter to message" or something along those lines rather than hard shut down?

Reply:
Checking for NULL on each value returned by `strtok_r` and logging a message if that is the case